runtime/runtime_test.go

```diff
@@ -479,3 +479,53 @@ func getTestRuntime(ctx context.Context, t *testing.T, logger logging.Logger) *R
 	}
 	return rt
 }
+
+func TestAddrWarningMessage(t *testing.T) {
```

Comment:
Tried to instead of having two separate tests to check for the warning, have one test instead. But happy to change this back to two individual tests. 👍